src/main/java/com/hackclub/hccore/commands/StatsCommand.java

```diff
@@ -25,28 +27,81 @@ public StatsCommand(HCCorePlugin plugin) {
 
     @Override
     public boolean onCommand(CommandSender sender, Command cmd, String alias, String[] args) {
-        // /stats
-        if (args.length == 0) {
+
+        // set extended to false by default
+        boolean extended = false;
+        // initialize specificStat variable
+        Statistic specificStat = null;
+
+        final List<Statistic> STATISTIC_NAMES = Arrays.asList(Statistic.values());
+
+        if ((args.length == 0) || (args[0].contains("extended"))) {
```

Comment:
The syntax is `/stats [player] extended`. If a player wants to get their own extended stats, they should run `/stats <their name> extended`.

Reply:
well i made it so someone could also do /stats extended and get their own

Reply:
i can remove that feature if you want

Reply:
the player can also do `/stats <player> extended`

Reply:
also [] means optional